docs/guides/latest-updates.mdx

```diff
@@ -29,6 +29,17 @@ Jump to [Qiskit SDK](#latest-qiskit-sdk) | [Qiskit Runtime client](#latest-qiski
 
 This section summarizes the recent enhancements and new features for IBM Quantum Platform.
 
+**10 April 2025**
+
+- Updated navigation to remove duplicated pages:
+  - The Administration section has been renamed "Access management" and focuses on managing users and access groups.
+  - There is only one page each for managing instances and workloads; there is no longer a separate page for administrators.  Instead, what you can view and manage on each page is dependent on your access and permissions.
+  - The analytics page (visible only to those with certain accesss) has been moved into the main navigation menu, to make it easier to find.
+- Launched two new QPUs: ibm_aachen (eu-de) and ibm_kingston (us-east)
+- Individuals who can view the analytics page can now filter the data to view information per plan.
+- Administrators can easily add multiple account members to an access group.
+- Premium plan instance administrators can set the instance allocation as a usage limit, so that usage in those instances cannot exceed the time alloted. See the [Set instance allocation limits](/guides/allocation-limits) guide for instructions.
+
```

Comment:
Please note, the sunset date for the classic version of IBM Quantum Platform has been set for July 1 2025. For more information, read the migration guide (link to migration guide in each respective platform)